extensions/data-workspace/src/dataworkspace.d.ts

```diff
@@ -115,4 +120,29 @@ declare module 'dataworkspace' {
 		 */
 		element: any;
 	}
+
+	export interface IProjectAction {
+		/**
+		 * id and display name of the project action
```

Comment:
Is "and display name" still true after the change to remove displayName?

Reply:
Actually, display name can be dropped here. Not really required.
Fixing it for next iteration.